apiserver/facades/agent/provisioner/provisioner.go

```diff
@@ -378,7 +387,7 @@ func (api *ProvisionerAPI) ContainerManagerConfig(ctx stdcontext.Context, args p
 	cfg := make(map[string]string)
 	cfg[container.ConfigModelUUID] = api.st.ModelUUID()
 
-	mConfig, err := api.m.ModelConfig(ctx)
+	mConfig, err := api.modelConfigService.ModelConfig(ctx)
```

Comment:
If the config is pulled from the service now, should it disappear from `api.m`?

Reply:
The `m` struct is the model that lives in the legacy state, so no point in removing stuff from it (we are removing it soon-ish). Furthermore, the full model hasn't still been migrated to dqlite so we can't fully remove this field from the `api`.

Reply:
Normally in these PR's we would do this. But this facade isn't using an interface for the model state so we can't remove it yet.